nixos/modules/services/x11/desktop-managers/kde5.nix

```diff
@@ -31,204 +31,224 @@ in
         '';
       };
 
+      extraPackages = mkOption {
+        type = types.listOf types.package;
+        default = [];
+        description = ''
+          KDE packages that need to be installed system-wide.
+        '';
+      };
+
     };
 
   };
 
 
-  config = mkIf (xcfg.enable && cfg.enable) {
+  config = mkMerge [
+    (mkIf (cfg.extraPackages != []) {
+      environment.systemPackages = [ (kde5.kdeWrapper cfg.extraPackages) ];
```

Comment:
> They are left there because they seem to not be wrapped in fact (it surprised me how few packages are wrapped).

Yes, only packages that are intended to be used as standalone applications need to be wrapped.